### ISSUE TITLE
Cannot use new "activate" command

### ISSUE THREAD

**Issue by bp88** [bartreardon/swiftDialog#282]
**Describe the bug**
A workflow we're trying to make use of is to have one main swiftDialog window that basically takes up the full screen using the `--blurscreen`. However, there are situations where we want to be able to show the user an error using a smaller dialog prompt.

On swiftDialog 2.2.1.4591, if I try to have two swiftDialog processes and then try to use `echo "activate:" >> "/var/tmp/dialog.log"` to bring one of the prompts to the front, it doesn't seem to actually bring it to the front. Both swiftDialog processes are pointing to different `commandfile`

This is a feature introduced in swiftDialog 2.2 as a result of this issue: #221 

As an example:
```
/bin/rm "/var/tmp/dialog.log"
/bin/rm "/var/tmp/dialog_err.log"

# Main dialog does showing user content
/usr/local/bin/dialog \
            --webcontent "file://path/to/index.html" \
            --title " " \
            --message none \
            --titlefont "size=14" \
            --messagefont "size=14" \
            --messagealignment "center" \
            --progress \
            --button1disabled \
            --quitkey q \
            --button1text "Please wait..." \
            --height 620 \
            --width 1000 \
            --infobox "**Computer Name:**TEST_COMPUTER" \
            --icon "/System/Library/CoreServices/CoreTypes.bundle/Contents/Resources/com.apple.macbookpro-14-2021-space-gray.icns" \
            --iconsize "150" \
            --helpmessage "Contact your IT administrator." \
	    --commandfile "/var/tmp/dialog.log" \
	    --blurscreen &

# Dialog prompt to alert user meant to show over the previous content screen
/usr/local/bin/dialog \
	--title none \
	--height "150" \
	--width "500" \
	--message "This Mac is on a 16% charge. Please plug in your Mac to prevent it from shutting down." \
	--titlefont "size=14" \
	--messagefont "size=14" \
	--messageposition "top" \
	--commandfile "/var/tmp/dialog_err.log" &

echo "activate:" >> "/var/tmp/dialog_err.log"
```

**Expected behavior**
I expect the command file that gets updated with the "activate" value will come to the forefront.

Currently, that is not happening.

**Screenshots**
Cannot capture screenshot that would help illustrate issue.

**Desktop (please complete the following information):**
 - macOS 13.4
 - swiftDialog 2.2.1.4591

**Additional context**
Is there support for multiple swiftDialog processes?

**Comment by bartreardon** [comment]
You cant activate a second dialog window above an existing dialog with blurscreen active. The blurred dialog will take precedence over all other windows, including other instances of swiftdialog.

**Comment by bp88** [comment]
Thanks for explaining. That makes sense.

Would you consider this as a potential enhancement or feature request? The reason I mention this is that a blurred dialog is one of the better looking ways to have content displayed to users. However, there are situations where you want to provide a user another dialog (in the example given above, it's an alert, but there may be other use cases). And at the moment, the only way to do that when you already have a blurred dialog is by simply creating another blurred dialog.

That process works, but there's an order of operations you need to be aware of where the dialog you want to be at the front needs to be the last blurred dialog so that it become the forefront dialog. Essentially, the ability to activate a dialog instance via the command file provides a more reliable method of ensuring the dialog becomes the forefront process.

Totally understand if this doesn't seem like it's worth the effort since there is a way to achieve this.

Btw thanks for sharing this handy app with the community.

**Comment by bartreardon** [comment]
Should be resolved in [v2.3](https://github.com/bartreardon/swiftDialog/releases/tag/v2.3.0) which has some extra logic to bring dialogs forward.

To use, launch your initial dialog with `--blurscreen`
launch subsequent dialogs with `--ontop` and specify a unique `--commandfile`.
after that, sending `activate:` to the second dialog windows will bring them in front of the dialog that has `--blurscreen` enabled.
